backend/app/api/routes/inquiries.py

```diff
@@ -36,13 +36,13 @@ def update_inquiry(*, session: SessionDep, inquiry_in: InquiryUpdate) -> Inquiry
         raise HTTPException(status_code=400, detail=str(e))
 
 
-@router.delete("/", response_model=InquiryDelete)
-def delete_inquiry(*, session: SessionDep, inquiry_in: InquiryDelete) -> InquiryDelete:
+@router.delete("/{inquiry_id}", response_model=Message)
```

Comment:
Question: did you run generate-client for the frontend to change the openAPI schema based on these changes?